server/config.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Flow Emulator
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+)
+
+type ConfigInfo struct {
+	ServiceKey string `json:"service_key"`
+	StartupDir string `json:"startup_dir"`
+}
+
+var _ http.Handler = &ConfigInfo{}
+
+func NewConfigInfo(emulatorServer *EmulatorServer) *ConfigInfo {
+	startupDir, _ := os.Getwd()
+
+	return &ConfigInfo{
+		ServiceKey: fmt.Sprintf("0x%s", emulatorServer.blockchain.ServiceKey().Address.Hex()),
```

Comment:
This should return the public key which you can compare no the address. Also the value should better be `ServicePublicKey`

Reply:
Actually we need to check the `ServicePrivateKey` which is  stored in flow.json